ratis-shell/src/main/java/org/apache/ratis/shell/cli/sh/command/LeaderElectionCommand.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ratis.shell.cli.sh.command;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.ratis.client.RaftClient;
+import org.apache.ratis.protocol.RaftClientReply;
+import org.apache.ratis.protocol.RaftPeer;
+import org.apache.ratis.protocol.RaftPeerId;
+import org.apache.ratis.shell.cli.RaftUtils;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class LeaderElectionCommand extends AbstractRatisCommand{
+  public static final String ADDRESS_OPTION_NAME = "address";
+  public static final String PAUSE_OPTION_NAME = "pause";
+  public static final String RESUME_OPTION_NAME = "resume";
+
+  /**
+   * @param context command context
+   */
+  public LeaderElectionCommand(Context context) {
+    super(context);
+  }
+
+  @Override
+  public String getCommandName() {
+    return "leaderElection";
```

Comment:
This command and the ElectCommand are closely related.  How about we have a base `election` command and then have the following subcommands?
- transfer
- pause
- resume

Reply:
will do